src/lib/configs/initConfig.ts

```diff
@@ -438,6 +438,58 @@ export function getReadonlyConfigInitFunction<
 
 const initializedConfigs = new Set<string>();
 
+function formatConfig(configString: string) {
+  const formattedConfig = configString
+    .trim()
+    .split("\n")
+    .flatMap((line, i, ar) => {
+      // If it's empty string - it was a newline - remove it
+      if (line.trim() === "") {
+        return [];
+      }
+
+      const maybePreviousLine = ar[i - 1];
+
```

Comment:
Can we check `undefined` once at a top of the function?

Reply:
I reworked this part so no it's less comments - more veriable names instead and one less if statement - but the result is the same